code/game/objects/items/weapons/RPD.dm

```diff
@@ -103,6 +103,7 @@ var/global/list/RPD_recipes=list(
 		"Gas Mixer"      = new /datum/pipe_info(PIPE_GAS_MIXER,			1, PIPE_TRIN_M),
 		"Thermal Plate"  = new /datum/pipe_info(PIPE_THERMAL_PLATE,		1, PIPE_UNARY),
 		"Injector"       = new /datum/pipe_info(PIPE_INJECTOR,     		1, PIPE_UNARY),
+		"Dual-Port Vent" = new /datum/pipe_info(PIPE_DP_VENT,			1, PIPE_UNARY),
```

Comment:
I know, and if you look at things like volume pumps you'll see they're also counted as unary, why?

if it's binary you only get 2 directions to pick from in the RPD, if it's unary you get 4.